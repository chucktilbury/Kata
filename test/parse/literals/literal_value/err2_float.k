;------------------------------------------------------------------------------
; Test if the scanner detects an error in a float
;------------------------------------------------------------------------------

var float some_name 123..12
