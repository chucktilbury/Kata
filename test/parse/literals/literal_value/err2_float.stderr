Syntax: /home/chuck/Src/Kata/test/parse/literals/literal_value/err2_float.k: 5: 25: expected module item or end of file but got literal float
Warning: /home/chuck/Src/Kata/test/parse/literals/literal_value/err2_float.k: 6: 1: module is empty
